CHANGELOG.rst

```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+v0.4.1
```

Comment:
I think you said this will be `0.5.0`, right? @TShapinsky

Reply:
I think that makes sense. It doesn't seem like we'll release anything else before `0.5.0`. I was mainly talking about the rest api changes since patches shouldn't have API breaking changes (though I think technically breaking changes are supposed to be major versions, maybe we'll do that after alfalfa gets to 1.0.0).